Hunting Queries/FileExecutionWithOneCharacterInTheName.yaml

```diff
@@ -0,0 +1,50 @@
+id: 299472c4-8382-4c5b-82d9-718cda193393
+name: Execution of File with One Character in the Name 
+description: |
+   'This query detects execution of files with one character in the name (e.g, a.exe, 7.ps1, g.vbs etc.). 
+   Normally files that are executed have more characters in the name and this can indicate a malicious file.
+   Ref: https://www.mandiant.com/resources/tactics-techniques-procedures-associated-with-maze-ransomware-incidents'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: SecurityEvents
+    dataTypes:
+      - SecurityEvent
+tactics:
+  - Execution
+relevantTechniques:
+  - T1059
+query: |
+  Event
+  | where EventLog == "Microsoft-Windows-Sysmon/Operational" and EventID==1
+  | parse EventData with * 'CommandLine">' CommandLine "<" *
+  | where CommandLine matches regex @'\\[a-zA-Z0-9]\.[a-zA-Z0-9]{2,5}["]{1}'
+  | parse EventData with * 'Image">' Image "<" *
+  | parse EventData with * 'ProcessGuid">' ProcessGuid "<" *
+  | parse EventData with * 'Description">' Description "<" *
+  | parse EventData with * 'OriginalFileName">' OriginalFileName "<" *
+  | parse EventData with * 'CommandLine">' CommandLine "<" *
+  | parse EventData with * 'CurrentDirectory">' CurrentDirectory "<" *
+  | parse EventData with * 'User">' User "<" *
+  | parse EventData with * 'LogonGuid">' LogonGuid "<" *
+  | parse EventData with * 'IntegrityLevel">' IntegrityLevel "<" *
+  | parse EventData with * 'Hashes">' Hashes "<" *
+  | parse EventData with * 'ParentProcessGuid">' ParentProcessGuid "<" *
+  | parse EventData with * 'ParentImage">' ParentImage "<" *
+  | parse EventData with * 'ParentCommandLine">' ParentCommandLine "<" *
+  | parse EventData with * 'ParentUser">' ParentUser "<" *
+  | summarize StartTime = min(TimeGenerated), EndTime = max(TimeGenerated) by EventID, Computer, User, ParentImage, ParentProcessGuid, ParentCommandLine, ParentUser, Image, ProcessGuid, CommandLine, Description, OriginalFileName, CurrentDirectory, Hashes
+  entityMappings:
```

Comment:
Indentation is broken here, remove the double space from here down to line 48

Reply:
Fixed